hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+- Set the RBW as big as possible, as long as the noise floor of SA should below the limit by 3 dB to 10dB.
+- Set the VBW to 3 times of RBW;
+- Set the trace mode to Max Hold;
+- Set the detector mode to Peak detector;
+- It can set the cable loss in fundamental and harmonics frequency points, this will count the cable loss in the display results;
+- It can set a marker table to display the signal strenghth at fundamental and harmonics.
+
+The following picture is a test example:
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\Tx-Power-and-Harmonics.png">  
+</div>
+
+
+# 5. Conducted Rx sensitivity measurement
+Sensitivity is basic test item for Rx performance. Usually it use a Signal Generator(SG) to generate a modulated signal and inject it into radio chip. Then the radio chip can demodulate the signal into data, the Railtest tool can use the data to count the BER. Once the BER critia is achieved, then the SG power level is called sensitivity.
+
+## 5.1. The Signal Generator settings
+At beginning of the test, it should configure SA to generate right modulated signal. Following steps will configure a GFSK modulated signal:
+
+- Set the SG to Custom Mode;
+- Set IQ data to PN9;
+- Set modulation type to 2-level FSK;
+- Set filter to 0.5Bt Gaussian type;
+- Set symble rate to the desired data rate, symble rate=data rate for 2GFSK modulation;
+- Set frequency deviation to desired frequency deviation; 
+- Set frequency to desired operating frequency;
+- RF: ON
+- MOD: ON
+- Tune signal strength level;
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\BER3.png">  
+</div>
+
+## 5.2. RailTest commands and test result
+The RailTest commands are listed below for Rx BER test: 
+  - :>rx 0
+  - :>Setchannel 0 
+  - :>SetBerconfig 1000
+  - :>BerRx 1
+  - :>......  
+  - :>Berstatus
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command;
+- Setchannel 0 will select a frequency points to test, it can also select another frequency point to test by changing the channel number;
+- SetBerconfig 1000 sets 1000 bytes(8000 bits) data to count the BER;
+- BerRx 1 sets the radio in BER RX mode;
+- Wait for a while. Then the Berstatus command will display the BER test result.
+
+<div align="center">
+  <img src="files\HW-RF-Test-Guide\BER4.png">  
+</div>
+
+When the PercentBitError rise up to 0.10, the SA's signal strength minus cable loss is the sensitivity 
+under 0.1% BER.
+
+# 6. Radiation test environment
+Radiation test should be performed in a RF shielded anechoic chamber. Most of the regulation test items can be done with radiation test in an anechoic chamber. Besides that some antenna radiation parameter can be tested with radiation test such as TRP(Total Radiation Power), TIS(Total Isotropic Sensitivity), 2D cuts, and 3D pattern.
```

Comment:
Change "a RF" to "an RF".  Insert spaces after TRP and TIS before (.

Reply:
Corrected.